tests/lib/managers/webhooks-test.js

```diff
@@ -12,6 +12,8 @@ var sinon = require('sinon'),
 
 var BoxClient = require('../../../lib/box-client');
 
+var assert = require('assert');
```

Comment:
This should go with the rest of the test library require() statements above.  Also, we use Chai for assertions, so this should be `require('chai').assert`